### PR TITLE
Update v1.5.x docs with latest changes

### PR DESCRIPTION
**Pull request by coderbyheart**

### CODE REVIEW

File: docs/azure/ContinuousIntegration.rst [hunk 1]
```diff
@@ -32,140 +30,191 @@ Prepare your Azure account
    The setup process in Azure is more complicated when compared to the :ref:`AWS continuous integration setup <aws-continuous-integration>`, since it involves many manual steps, which cannot be automated.
    If you have ideas to simplify the process, `please provide your input <https://github.com/NordicSemiconductor/asset-tracker-cloud-azure-js/issues/1>`_.
 
-Create a secondary tenant (Azure Active Directory B2C)
-======================================================
+Login and create the resource group
```
Comment (peknis) [inline docs/azure/ContinuousIntegration.rst]:
```suggestion
Log in and create the resource group
```

File: docs/azure/ContinuousIntegration.rst [hunk 2]
```diff
@@ -32,140 +30,191 @@ Prepare your Azure account
...
+===================================
+
+To create the resource group for the CI resources, complete the following steps:
+
+1. Login to the shell:
```
Comment (peknis) [inline docs/azure/ContinuousIntegration.rst]:
```suggestion
1. Log in to the shell:
```

File: docs/azure/ContinuousIntegration.rst [hunk 3]
```diff
@@ -32,140 +30,191 @@ Prepare your Azure account
...
    .. code-block:: bash
 
-      export B2C_TENANT="<Primary domain>" # For example, "nrfassettrackerciusers"
+       az account set --subscription $SUBSCRIPTION_ID 
+       # Verify that it is set to default
+       az account list --output table
 
-#. Link this Azure AD B2C tenant to the subscription for CI by following the `Billing guide <https://docs.microsoft.com/en-us/azure/active-directory-b2c/billing#link-an-azure-ad-b2c-tenant-to-a-subscription>`_.
+#. Choose a resource group name for the solution and export it as ``RESOURCE_GROUP``.
+   In this example, we use ``nrfassettrackerci`` as the resource group name.
```
Comment (peknis) [inline docs/azure/ContinuousIntegration.rst]:
```suggestion
   In this example, ``nrfassettrackerci`` is used as the resource group name.
```

File: docs/azure/ContinuousIntegration.rst [hunk 4]
```diff
@@ -32,140 +30,191 @@ Prepare your Azure account
...
-Create the Azure Active Directory B2C application
--------------------------------------------------
+   .. code-block:: bash
 
-To create the Azure Active Directory B2C application, complete the following steps:
+      export RESOURCE_GROUP="nrfassettrackerci"
+    
+#. Choose a name for the solution and export it as ``APP_NAME``.
+   Use a short name (not more than 16 characters) composed of numbers and lower-case letters only.
+   In this example, we use ``nrfassettrackerci`` as the application name.
```
Comment (peknis) [inline docs/azure/ContinuousIntegration.rst]:
```suggestion
   In this example, ``nrfassettrackerci`` is used as the application name.
```

File: docs/azure/ContinuousIntegration.rst [hunk 5]
```diff
@@ -32,140 +30,191 @@ Prepare your Azure account
...
+   .. parsed-literal::
+      :class: highlight
 
-   * Name - Cat Tracker Web App.
-   * Redirect URI (make sure to select SPA) - ``https://nrfassettrackerciapp.z16.web.core.windows.net/`` (Choose a name that fits your project instead of ``nrfassettrackerciapp`` since ``nrfassettrackerciapp`` is globally unique).
+      export APP_NAME="nrfassettrackerci"
 
-#. Export the ``Application (client) ID`` and the ``Directory (tenant) ID`` of the created Active Directory B2C App registration into the ``APP_REG_CLIENT_ID`` and ``B2C_TENANT_ID`` parameters:
+#. Choose a resource group and a name for the Device Update instance and export it as ``ADU_RESOURCE_GROUP`` and ``ADU_INSTANCE_NAME``.
+   There is an undocumented limit of 2 instances per ADU account, so the CI should be run against a separate Device Update instance.
```
Comment (peknis) [inline docs/azure/ContinuousIntegration.rst]:
```suggestion
   There is an undocumented limit of two instances per ADU account, so the CI should be run against a separate Device Update instance.
```

File: docs/azure/ContinuousIntegration.rst [hunk 6]
```diff
@@ -32,140 +30,191 @@ Prepare your Azure account
...
+   In this example, we use ``nRFAssetTrackerADUCI`` as the resource group and Device Update instance name.
```
Comment (peknis) [inline docs/azure/ContinuousIntegration.rst]:
```suggestion
   In this example, ``nRFAssetTrackerADUCI`` is used as the resource group and Device Update instance name.
```

File: docs/azure/ContinuousIntegration.rst [hunk 7]
```diff
@@ -32,140 +30,191 @@ Prepare your Azure account
...
 
-      export APP_REG_CLIENT_ID="<Application (client) id>"
-      export B2C_TENANT_ID="<Directory (tenant) ID>"
+      # add to .envrc
+      export ADU_RESOURCE_GROUP="nRFAssetTrackerADUCI"
+      export ADU_INSTANCE_NAME="nRFAssetTrackerADUCI"
 
-#. For enabling the test-runner to programmatically login users, enable `the resource owner password credentials (ROPC) flow <https://docs.microsoft.com/EN-US/azure/active-directory-b2c/configure-ropc?tabs=app-reg-ga>`_ with the following settings:
+#. Configure your preferred location (you can list the locations using ``az account list-locations``) and export it on the environment variable ``LOCATION``.
+   In this example, we use ``northeurope`` as the location name.
```
Comment (peknis) [inline docs/azure/ContinuousIntegration.rst]:
```suggestion
   In this example, ``northeurope`` is used as the location name.
```

File: docs/azure/ContinuousIntegration.rst [hunk 8]
```diff
@@ -32,140 +30,191 @@ Prepare your Azure account
...
-       az account list --output table
+   .. parsed-literal::
+      :class: highlight
 
-#. Create the CI credentials:
+      export APP_REG_CLIENT_ID="*Application (client) id*"
+      export B2C_TENANT_ID="*Directory (tenant) ID*"
 
-   .. code-block:: bash
+#. For enabling the test-runner to programmatically login users, enable `the resource owner password credentials (ROPC) flow <https://docs.microsoft.com/EN-US/azure/active-directory-b2c/configure-ropc?tabs=app-reg-ga>`_ with the following settings on the Active Directory B2C:
```
Comment (peknis) [inline docs/azure/ContinuousIntegration.rst]:
```suggestion
#. For enabling the test-runner to programmatically log in users, enable `the resource owner password credentials (ROPC) flow <https://docs.microsoft.com/EN-US/azure/active-directory-b2c/configure-ropc?tabs=app-reg-ga>`_ with the following settings on the Active Directory B2C:
```

File: docs/azure/ContinuousIntegration.rst [hunk 9]
```diff
@@ -32,140 +30,191 @@ Prepare your Azure account
...
-       location=${LOCATION:-northeurope} \
-       appRegistrationClientId=$APP_REG_CLIENT_ID \
-       b2cTenant=$B2C_TENANT \
-       b2cFlowName=B2C_1_developer
+      export B2C_CLIENT_SECRET="*client secret*"
 
-#. Publish the functions:
+#. Link this Azure AD B2C tenant to the subscription for CI by following the `Billing guide <https://docs.microsoft.com/en-us/azure/active-directory-b2c/billing#link-an-azure-ad-b2c-tenant-to-a-subscription>`_.
+
+Setup continuous integration on GitHub
```
Comment (peknis) [inline docs/azure/ContinuousIntegration.rst]:
```suggestion
Set up continuous integration on GitHub
```

File: docs/azure/GettingStarted/Deploy.rst [hunk 10]
```diff
@@ -30,6 +33,16 @@ Deploy the solution to your account
       # add to .envrc
       export APP_NAME="nrfassettracker"
 
+#. Choose a resource group and a name for the Device Update instance and export it as ``ADU_RESOURCE_GROUP`` and ``ADU_INSTANCE_NAME``.
+   There is a limit of `2 instances per subscription <https://docs.microsoft.com/en-us/azure/iot-hub-device-update/device-update-resources#device-update-instance>`_, so the ADU instance needs to be shared between multiple IoT hubs.
```
Comment (peknis) [inline docs/azure/GettingStarted/Deploy.rst]:
```suggestion
   There is a limit of `two instances per subscription <https://docs.microsoft.com/en-us/azure/iot-hub-device-update/device-update-resources#device-update-instance>`_, so the ADU instance needs to be shared between multiple IoT hubs.
```

File: docs/azure/GettingStarted/Deploy.rst [hunk 11]
```diff
@@ -30,6 +33,16 @@ Deploy the solution to your account
...
+   In this example, we use ``nRFAssetTrackerADU`` as the resource group and Device Update instance name.
```
Comment (peknis) [inline docs/azure/GettingStarted/Deploy.rst]:
```suggestion
   In this example, ``nRFAssetTrackerADU`` is used as the resource group and Device Update instance name.
```

File: docs/azure/GettingStarted/Deploy.rst [hunk 12]
```diff
@@ -30,6 +33,16 @@ Deploy the solution to your account
...
+
+   .. code-block:: bash
+
+      # add to .envrc
+      export ADU_RESOURCE_GROUP="nRFAssetTrackerADU"
+      export ADU_INSTANCE_NAME="nRFAssetTrackerADU"
+
 #. Configure your preferred location (you can list the locations using ``az account list-locations``) and export it on the environment variable ``LOCATION``.
    In this example, we use ``northeurope`` as the location name.
```
Comment (peknis) [inline docs/azure/GettingStarted/Deploy.rst]:
```suggestion
   In this example, ``northeurope`` is used as the location name.
```

File: docs/firmware/azure/BuildingUsingDocker.rst [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+
+nRF9160 DK (PCA10090)
+=====================
+
+.. code-block:: bash
+
+    docker run --rm -v ${PWD}:/workdir/ncs/firmware asset-tracker-firmware-docker /bin/bash -c 'cd /workdir/ncs/firmware; west build -p always -b nrf9160dk_nrf9160ns -- -DOVERLAY_CONFIG="overlay-debug.conf;asset-tracker-cloud-firmware-azure.conf"'
+    ls -la build/zephyr/merged.hex
+
+Location of the HEX file
```
Comment (peknis) [inline docs/firmware/azure/BuildingUsingDocker.rst]: This section could be on the same level as the devices -> ====

File: docs/firmware/azure/BuildingUsingLocalSystem.rst [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+
+nRF9160 DK (PCA10090)
+=====================
+
+.. code-block:: bash
+
+    west build -p always -b nrf9160_pca10090ns
+
+Location of the HEX file
+************************
```
Comment (peknis) [inline docs/firmware/azure/BuildingUsingLocalSystem.rst]: This section could be at the same level as the devices -> ====

**Review by peknis — CHANGES_REQUESTED** [review]

File: docs/azure/ContinuousIntegration.rst [hunk 15]
```diff
@@ -32,140 +30,191 @@ Prepare your Azure account
...
 
-Create the Azure Active Directory B2C application
--------------------------------------------------
+   .. code-block:: bash
 
-To create the Azure Active Directory B2C application, complete the following steps:
+      export RESOURCE_GROUP="nrfassettrackerci"
+    
+#. Choose a name for the solution and export it as ``APP_NAME``.
+   Use a short name (not more than 16 characters) composed of numbers and lower-case letters only.
```
Comment (umapraseeda) [inline docs/azure/ContinuousIntegration.rst]:
```suggestion
   Use a short name (not more than 16 characters) composed of numbers and lowercase letters only.
```

File: docs/azure/ContinuousIntegration.rst (hunk 5 above)
Comment (umapraseeda) [inline docs/azure/ContinuousIntegration.rst]:
```suggestion
   There is an undocumented limit of two instances per ADU account, so the CI must be run against a separate Device Update instance.
```

**Review by umapraseeda — APPROVED** [review]

**Review by peknis — APPROVED** [review]

**Pull request merged**